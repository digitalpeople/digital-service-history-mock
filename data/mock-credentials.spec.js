import {
  mockUsername,
  mockPassword,
} from './mock-credentials';

jest.mock('../app/utils/hash', () => ({
  hash: jest.fn(value => `mock-hash-${value}`),
}));

describe('mockUsername', () => {
  it('should return mock value', () => {
    expect.assertions(1);

    const actual = mockUsername;

    expect(actual).toMatchSnapshot();
  });
});

describe('mockPassword', () => {
  it('should return mock value', () => {
    expect.assertions(1);

    const actual = mockPassword;

    expect(actual).toMatchSnapshot();
  });
});
