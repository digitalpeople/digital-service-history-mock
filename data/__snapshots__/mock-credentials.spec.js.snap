// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mockPassword should return mock value 1`] = `"mock-hash-12345"`;

exports[`mockUsername should return mock value 1`] = `"test"`;
